.appveyor.yml

```diff
@@ -0,0 +1,13 @@
+version: "#{build}"
+pull_requests:
+  do_not_increment_build_number: true
+configuration:
+  - Release
+clone_depth: 5
+build_script:
+  - cmd: call .appveyor\appveyor_helper.bat
```

Comment:
@arungrover We just got one build fail where we can see that what happens if any of PBS deps fails... see [here](https://ci.appveyor.com/project/hirenvadalia/pbspro/build/%23228)